README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portafolio
-This project shows all my experience as a software Developer
+It is my portfolio, experience has a software developer
+
+> Description the project.
+- **This portfolio show my experience as a software developer.**
+> ## Objectives
+- Build a Mobile First portafolio
+- Follow the best practices of adabtative content and Mobile First
+- Setup an skeleton 
+- Use flexBox
+- Follow GitHub flow.
+- Open pull requests in GitHub.
+
+
+## Built With
+- HTML
+- CSS
+
+## Live Demo (if available)
+we don't have a demo yet
+
+
```

Comment:
- [ ] Kindly consider removing sections from your readme that aren't being used. They bring about an incomplete README feeling.